mmv1/products/iap/Settings.yaml

```diff
@@ -0,0 +1,277 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Settings'
+description: |
+  gcloud iap settings - manage IAP settings
+base_url: '{{name}}:iapSettings'
+self_link: '{{name}}:iapSettings'
+create_verb: :PATCH
+update_verb: :PATCH
+update_url: '{{name}}:iapSettings'
+import_format: ['{{name}}/iapSettings']
+# Skipping the sweeper because instances will be deleted during cluster sweeps
+skip_sweeper: true
+
```

Comment:
Can you try to add `id_format: '{{name}}/iapSettings'` ?

Reply:
Sure, I will try that, should I replace it with `import_format` or add below it

Reply:
Both `import_format` and `id_format` are needed.